tools/createGenesis/generateGenesis.ts

```diff
@@ -29,7 +29,7 @@ import {ProxyAdmin} from "../../typechain-types";
 import {Addressable} from "ethers";
 import "../../deployment/helpers/utils";
 
-const deployParameters = require("./deploy_parameters.json");
+const genesisParameters = require("./genesis_parameters.json");
```

Comment:
actually what this tool did, was to migrate a previous genesis ( non-LxLY genesis, to an LxLY genesis, this was to the etrog fork ^^) 
So the correct name should be upgrade_pre_etrog_genesis